### PR TITLE
Docs fix

### PR DESCRIPTION
**Pull request by FreZZZeR**
### Description

Docs fix for function Set-Attachement
Added and fixed Suppress Exclusion in Invoke-WebRequest, Set-TlsLevel

### Motivation and Context

Fix bugs in documentation
Fix Suppress Exclusion in Invoke-WebRequest, Set-TlsLevel

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] My code follows the code style of this project.
- [ ] I have added Pester Tests that describe what my changes should do.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FreZZZeR** [comment]
@lipkau , Hello!
In Azure Pipeline we have 2 errors when testing ConfluencePS functions

    Context /ConfluencePS/Private/Invoke-WebRequest.ps1
      [-] passes PSAvoidOverwritingBuiltInCmdlets 116ms
        Expected $null, but got @(1, 161).
        70:                     $BadLines | Should -Be $null
        at <ScriptBlock>, /home/vsts/work/1/s/Release/Tests/PSScriptAnalyzer.Tests.ps1: line 70

    Context /ConfluencePS/Private/Set-TlsLevel.ps1
      [-] passes PSReviewUnusedParameter 26ms
        Expected $null, but got @(6, 9).
        70:                     $BadLines | Should -Be $null
        at <ScriptBlock>, /home/vsts/work/1/s/Release/Tests/PSScriptAnalyzer.Tests.ps1: line 70


I tried using both functions on Windows 10 2004 Preview on WSL (Ubuntu 18.04, Ubuntu 20.04), and on virtual machine with Ubuntu 18.04 LTS
No one error took.
Can You explain how is your test works?

**Comment by FreZZZeR** [comment]
I did not notice error messages.
_PSAvoidOverwritingBuiltInCmdlets_
Can we rename function Invoke-WebRequest to pass that test?
_PSReviewUnusedParameter_
 I repaired it in function Set-TlsLevel.ps1 setting Mandatory = $true

**Comment by FreZZZeR** [comment]
@lipkau,  I fixed all error. 
I have no experience in team development using GIT.
Can you review my fixes?

**Comment by lipkau** [comment]
Sorry. I missed the other PR.

File: ConfluencePS/Private/Set-TlsLevel.ps1 [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 
-        [Parameter(Mandatory, ParameterSetName = 'Revert')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'Revert')]
         [Switch]$Revert
     )
 
     begin {
-        switch ($PSCmdlet.ParameterSetName) {
-            "Set" {
+        switch ($true) {
```
Comment (lipkau) [inline ConfluencePS/Private/Set-TlsLevel.ps1:16]: Why are you changing this?
I don't see the benefit.

**Review by lipkau — CHANGES_REQUESTED** [review]

File: ConfluencePS/Private/Set-TlsLevel.ps1 (hunk 1 above)
Comment (FreZZZeR) [reply ConfluencePS/Private/Set-TlsLevel.ps1:16]: Yes. This is need for pass psscriptanalyzer tests

Here it is:
https://dev.azure.com/AtlassianPS/ConfluencePS/_build/results?buildId=462&view=logs&j=b80c90c8-f62d-51c1-0986-3bb8359d9b6f&t=2b940409-1a40-5f39-a766-f5d254a60ca2

Text:
    Context /ConfluencePS/Private/Set-TlsLevel.ps1
      [-] passes PSReviewUnusedParameter 27ms
        Expected $null, but got @(6, 9).
        70:                     $BadLines | Should -Be $null
        at <ScriptBlock>, /home/vsts/work/1/s/Release/Tests/PSScriptAnalyzer.Tests.ps1: line 70

Comment (FreZZZeR) [reply ConfluencePS/Private/Set-TlsLevel.ps1:16]: If You look my comments in the PR and commit messages, You will understand me.
Thanks!

Comment (FreZZZeR) [reply ConfluencePS/Private/Set-TlsLevel.ps1:16]: Have I to rewrite that with [Diagnostics.CodeAnalysis.SuppressMessageAttribute()]?

Comment (lipkau) [reply ConfluencePS/Private/Set-TlsLevel.ps1:16]: No.
I didn't know that new rule yet.

Great job.
Thank you for the changes

**Review by lipkau — DISMISSED** [review]

File: ConfluencePS/Private/Set-TlsLevel.ps1 (hunk 1 above)
Comment (FreZZZeR) [reply ConfluencePS/Private/Set-TlsLevel.ps1]: I think You was right, so I repair branch for passing test without changing switch statement

**Comment by FreZZZeR** [comment]
Finally, documentation was fixed. Tests can be passed.

**Review by FreZZZeR — COMMENTED** [review]
All changes has been reviewed.

**Review by lipkau — APPROVED** [review]

**Pull request merged**